src/main.cpp

```diff
@@ -1937,13 +1953,82 @@ void setup() {
         .ptr = (void *)&standbyModeTime
     };
 
+#if (ONLYPIDSCALE == 1 || BREWMODE == 2)
+    editableVars["TARE_ON"] = {
+        .displayName = F("Tare"),
+        .hasHelpText = false,
+        .helpText = "",
+        .type = kUInt8,
+        .section = sScaleSection,
+        .position = 29,
+        .show = [] { return false; },
+        .minValue = 0,
+        .maxValue = 1,
+        .ptr = (void *)&scaleTareOn
+    };
+
+    editableVars["CALIBRATION_ON"] = {
+        .displayName = F("Calibration"),
+        .hasHelpText = false,
+        .helpText = "",
+        .type = kUInt8,
+        .section = sScaleSection,
+        .position = 30,
+        .show = [] { return false; },
+        .minValue = 0,
+        .maxValue = 1,
+        .ptr = (void *)&scaleCalibrationOn
+    };
+
+        editableVars["SCALE_KNOWN_WEIGHT"] = {
+        .displayName = F("Known weight in g"),
+        .hasHelpText = false,
+        .helpText = "",
+        .type = kFloat,
+        .section = sScaleSection,
+        .position = 31,
+        .show = [] { return true; },
+        .minValue = 0,
+        .maxValue = 2000,
+        .ptr = (void *)&scaleKnownWeight
+    };
+
+        editableVars["SCALE_CALIBRATION"] = {
+        .displayName = F("Calibration factor scale 1"),
+        .hasHelpText = false,
+        .helpText = "",
+        .type = kFloat,
+        .section = sScaleSection,
+        .position = 32,
+        .show = [] { return true; },
+        .minValue = -100000,
+        .maxValue = 100000,
+        .ptr = (void *)&scaleCalibration
+    };
+
+    #if SCALE_TYPE == 0
+        editableVars["SCALE2_CALIBRATION"] = {
+        .displayName = F("Calibration factor scale 2"),
+        .hasHelpText = false,
+        .helpText = "",
+        .type = kFloat,
+        .section = sScaleSection,
+        .position = 32,
+        .show = [] { return true; },
+        .minValue = -100000,
+        .maxValue = 100000,
+        .ptr = (void *)&scale2Calibration
+    };
+    #endif
```

Comment:
This one has to stay though, right? Otherwise we always show both on the website, which is not great either...